src/core/vdom/patch.js

```diff
@@ -106,6 +106,7 @@ export function createPatchFunction (backend) {
 
   function isUnknownElement (vnode, inVPre) {
     return (
+      !(typeof window !== 'undefined' && window.customElements && window.customElements.get(vnode.tag)) &&
```

Comment:
Let's cache checks for `customElements` so that we can just do `customElements && customElements.get(vnode.tag)`. Also can use `inBrowser` from utils instead of `typeof window`

Reply:
I have updated my code. I use `inBrowser` to check the environment and cache the `customElement` in a variable.